index.html

```diff
@@ -1455,12 +1456,54 @@ <h3>
           <p>
             Given {{GamepadEffectParameters}} |params:GamepadEffectParameters|,
             a <dfn>valid dual-rumble effect</dfn> must have a valid
-            duration, a valid startDelay, and both the
+            {{GamepadEffectParameters/duration}}, a valid {{GamepadEffectParameters/startDelay}}, and both the
             {{GamepadEffectParameters/strongMagnitude}} and the
             {{GamepadEffectParameters/weakMagnitude}} must be in the range
             `[0,1]`.
           </p>
         </dd>
+        <dt>
+          <dfn data-for="GamepadHapticEffectType">"trigger-rumble" effect
+          type</dfn>
+        </dt>
+        <dd>
+          <p>
+            {{GamepadHapticEffectType/"trigger-rumble"}} describes a haptics
+            configuration with a vibration motor in each of the bottom front buttons
+            of a [=Standard Gamepad=] (buttons with [=canonical indexes=] 6 and 7) in addition to the two handle motors used for 
```

Comment:
Both "indexes" and "indices" are valid plurals for "index", but we should choose one and use it consistently within the spec. "Indices" is already used elsewhere in the spec so I think we should prefer `[=canonical indices=]` here.